src/main/java/lotto/model/LottoNumbers.java

```diff
@@ -0,0 +1,95 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import lotto.util.RandomUtil;
+import lotto.util.SplitUtil;
+
+public class LottoNumbers {
+	public final static int LOTTO_NUMBERS_SIZE = 6;
+	private final static String STARTING_BRACKET = "[";
+	private final static String END_BRACKET = "]";
+	private final static String SEPARATOR_BRACKET = " ,";
+	private final static int START_INDEX_CALCULATE_NUMBER = 0;
+	private final static int LAST_INDEX_CALCULATE_NUMBER = 1;
+
+	public final List<LottoNumber> lottoNumberList;
+
+	public LottoNumbers() {
+		lottoNumberList = generateLottoNumberList();
+	}
+
+	public LottoNumbers(List<LottoNumber> lottoNumberList) {
+		this.lottoNumberList = lottoNumberList;
+	}
```

Comment:
Collection에서 제공하는 unmodifiableList를 활용해보는 건 어떨까요?
일급 컬렉션도 좀 더 불변 객체에 가까워질 수 있습니다 :)

```suggestion
	public LottoNumbers(List<LottoNumber> lottoNumberList) {
		this.lottoNumberList = Collections.unmodifiableList(lottoNumberList);
	}
```